Troubleshooting.md

```diff
@@ -50,4 +51,18 @@
      NOTE: You should see two persistent volumes as shown above, `registry-volume` is by default present for you and `manual-pipeline-run-pvc` 
      is supposed to be applied as manual pre-requisite step before running any pipelinerun. Follow steps [here](https://github.com/kabanero-io/kabanero-pipelines/blob/master/README.md#create-a-persistent-volume)
     
-   
+2. I get the below error while running the building step of any kabanero pipeline if I forget to do below three things as pre-requisite
```

Comment:
I suggest this text:  I see the error below while running the building step of a Kabanero Pipeline.  What causes this error?